src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -271,6 +274,107 @@ public void registerUserForCourse(User user, Course course) {
         log.info("User " + user.getLogin() + " has successfully registered for course " + course.getTitle());
     }
 
+    /**
+     * Fetches Course Management data from repository and returns a list of Course DTOs
+     *
+     * @param isOnlyActive Whether or not to include courses with a past endDate
+     * @return A list of Course DTOs for the course management overview
+     */
+    public List<Course> getAllCoursesForOverview(Boolean isOnlyActive) {
```

Comment:
```suggestion
    public List<Course> getAllCoursesForOverview(Boolean onlyConsiderActiveCourses) {
```

Reply:
Renamed it to `onlyActive` since this is the term we usually use in our code